cmd/simulator/load/loader.go

```diff
@@ -129,5 +136,10 @@ func ExecuteLoader(ctx context.Context, config config.Config) error {
 		return err
 	}
 	log.Info("Tx agents completed successfully.")
+
+	// Start a prometheus server to expose individual tx metrics
+	http.Handle("/metrics", promhttp.HandlerFor(reg, promhttp.HandlerOpts{Registry: reg}))
+	log.Info("localhost:8082/metrics")
+	http.ListenAndServe(":8082", nil)
```

Comment:
If we start this here, doesn't this mean that the metrics will only be exposed after the agents have already completed?

Reply:
Yes, that's what we want so we get all the data?

Reply:
Also listening is a blocking operation so it made sense to put it last